mobile/README.md

```diff
@@ -0,0 +1,20 @@
+# [Wolox](https://wolox.co) Mobile tech guides and standards
+
+Under every technology folder mentioned [below](#technologies-index), you will find a README describing:
+
+- **Kickoff Guides**: Step by step guide to make a kickoff to start building an app.
+- **Open Source**: A list of the open source libraries developed at Wolox.
+- **Useful Documentation**: Good practices, talks, useful guides and highlights.
+- **Posts**: Posts written for the [Wolox Engineering Blog](http://eng.wolox.co) for the given technology.
+
+## Mobile technologies we develop at [Wolox](https://wolox.co)
+
+- [React Native](../react-native/README.md)
+
+- [iOS](../iOS/README.md)
+
+- [Android](../android/README.md)
+
+## General Useful Documentation
+
+- [Mobile conventions naming](./docs/naming/README.md)
```

Comment:
I'd copy this under every tech's `General Useful Documentation` so we don't add`mobile` as tech on the main `README.md` (which is confusing since we have Android, RN, iOS and then mobile, which includes all of them).

Reply:
Yeah, better this way.